resources/event-bus/values.yaml

```diff
@@ -6,8 +6,8 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   event_bus:
-    dir: develop/
-    version: 524c6211
+    dir: pr/
+    version: PR-4344
```

Comment:
This needs to be fixed, right?

Reply:
No. This is correct.

Reply:
@marcobebway This needs to be changed before merging to master, no?
WHat's the plan here?

Reply:
@montaro please read this https://github.com/kyma-project/community/blob/master/sig-and-wg/sig-core/proposals/update-code-and-chart-in-one-pr.md.

Reply:
Thanks!